client/src/components/Header/Header.js

```diff
@@ -1,44 +1,67 @@
-import React from "react";
+import React, { useState } from "react";
 import {
     Toolbar,
     AppBar,
     Button,
-    TextField
+    TextField,
+    Grid,
+    Box
 } from "@material-ui/core";
 import { NavLink } from "react-router-dom";
 import Logo from "../common/Logo/Logo";
 import { useStyles } from './styles';
 
 const Header = () => {
     const classes = useStyles();
+    const products = [];
+    const [seachInput, setSearchInput] = useState('');
+    const filteredProducts = () => {
```

Comment:
(Я понимаю, что это заглушка до момента, когда подключимся к серверу, но всё же)
Я бы назвал эту функцию как-то так
```suggestion
    const searchforProducts = () => {
```
или
```suggestion
    const productSearchHandler = () => {
```

Потому что на следующей строке почти такая же по названию переменная, это немного сбивает с толку

Reply:
fixed